gemini/getting-started/intro_gemini_pro_python.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>Some of the modules here may be not necessary for Gemini pro.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Good point, removed unnecessary ones.</p>